src/toggle/__tests__/toggle.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('Toggle', () => {
     const refine = jest.fn();
     const fixture = render({ refine });
 
-    const toggle = fixture.debugElement.nativeElement.querySelector('li');
+    const toggle = fixture.debugElement.nativeElement.querySelector('label');
     toggle.click();
 
     expect(refine).toHaveBeenCalled();
```

Comment:
We can make this test a bit stricter.

```suggestion
     expect(refine).toHaveBeenCalledTimes(1);
```